### DISCUSSION TITLE
What do you use it for?

### DISCUSSION THREAD

**Discussion by jtmuller5** [jtmuller5/The-HustleGPT-Challenge#21] [Polls]
Its clear that ChatGPT can be used for everything from brainstorming to writing copy to writing code to just about everything else you can imagine

**Comment by Oricrypto** [comment]
I'm prompt engeneering the free version of GPT tô se How Far i can Go into the AI co founder Challenge,  while Bing's gpt4 wont follow the HustleGPT Idea,  It knows about it
